.github/workflows/build-images-action.yml

```
@@ -23,6 +23,19 @@ jobs:
       QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
       QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  build_ironic_cs10:
+    name: Build Ironic container image based on CentOS Stream 10
+    env:
+      IMAGE_BUILD_ARGS: ['BASE_IMAGE=quay.io/centos/centos:stream10']
```

```suggestion
      IMAGE_BUILD_ARGS:
      - 'BASE_IMAGE=quay.io/centos/centos:stream10'
```
This list format is more readable, and especially if we add anything else there.